store/sql_store.go

```diff
@@ -110,6 +111,14 @@ func initConnection() *SqlStore {
 		}
 	}
 
+	if len(utils.Cfg.SqlSettings.DataSourceSearchReplica) == 0 {
+		sqlStore.searchReplica = sqlStore.master
```

Comment:
@crspeller should I use a separate round robin always (I think this probably makes more sense), or use the read-replica one when there are no search replicas set?